client/sass/application.scss

```diff
@@ -1,3 +1,101 @@
 $govuk-global-styles: true;
 
 @import "node_modules/govuk-frontend/govuk/all";
+
+///// INTERRUPT PATTERN - FLASH CARD //////
+.flash-card {
+  margin-top: -1em;
+  padding: 30px;
+  margin-bottom: 30px;
+  background: #005EA5;
+  color: #FFF;
+
+  header {
+    background-color: inherit;
+  }
+
+  p {
+    color: #FFF;
+  }
+
+  ul {
+    color: #FFF;
+
+    h2 {
+      margin: 0;
+      margin-bottom: 0;
+      padding: 0;
+      color: #FFF;
+      font-size: 28px;
+    }
+
+    li {
+      color: #FFF;
+    }
+  }
+
+  h1 {
+    color: #FFF;
+    &:first-of-type {
+      margin-top: 0 !important;
+    }
+  }
+
+  a {
+    color: #FFF;
+    &:visited, &:hover, &:active {
+      color: #FFF; }
+    }
+    details {
+      // margin-bottom: 1em;
+      summary {
+        color: #FFF;
+      }
+      .panel {
+        border-left-color: #FFF;
+      }
+    }
+
+    button {
+      color: #FFF;
+      &:visited, &:hover, &:active {
+        color: #FFF; }
+      }
+      details {
+        // margin-bottom: 1em;
+        summary {
+          color: #FFF;
+        }
+        .panel {
+          border-left-color: #FFF;
+        }
+      }
+  }
+
+  @media (min-width: 769px) {
+    .flash-card {
+      // margin-top: 1.25em;
+      padding: 50px;
+      margin-bottom: 30px;
+
+    }
+  }
+
+
+.button-white {
```

Comment:
Use BEM naming conventions.

For this, should be:

.button--white {
...
}

Reply:
Alternatively you could consider 'flash-card' to be the block and the button could be an element with it,

i.e. 

.flash-card__button--white {
// etc
}}

Depends on whether you think a white button might appear elsewhere or if it is only ever going to be part of a 'flash-card' component.